cobalt/build/configs/cobalt.gni

```diff
@@ -15,4 +15,5 @@
 # is_cobalt is just declared here and potentially overridden while running gn gen.
 declare_args() {
   is_cobalt = false
+  cobalt_is_release_build = false
 }
```

Comment:
After the declare_args() block, add `assert(!cobalt_is_release_build || is_official_build, "Release builds are only available in official builds.")`